schemas/monocle/protob/metric.proto

```diff
@@ -22,7 +22,7 @@ message ListResponse {
 }
 
 message Trend {
-  uint32 interval = 1;
```

Comment:
I don't think using a protobuf enum to serialize trend interval is much better than encoding the data using a string. If we were using [typical](https://github.com/stepchowfun/typical#typical-data-interchange-with-algebraic-data-types), then perhaps it would be more practical to use a more correct representation, such as a sum type for the interval value.

In anycase, it may be easier to implement a custom string encoder/decoder to convert the user input form to the correct value, than directly providing the data type required by the backend.

Reply:
Ok so I set this as resolve as we use the TimeRange parser.